lib/uplink/uplink.go

```diff
@@ -118,7 +118,7 @@ func NewUplink(ctx context.Context, cfg *Config) (_ *Uplink, err error) {
 	defer mon.Task()(&ctx)(&err)
 
 	ident, err := identity.NewFullIdentity(ctx, identity.NewCAOptions{
-		Difficulty:  9,
+		Difficulty:  0,
```

Comment:
I'm wondering why it's not breaking version checks?

Reply:
yeah i'm confused too

Reply:
This is the minimum difficulty, no?